File: UVa/300/369.py
from math import factorial

while True:
    n,k = map(int, input().split())
    if n == 0 and k == 0:
        exit()

    print("%d things taken %d at a time is %d exactly." % (n, k, factorial(n) // factorial(k) // factorial(n - k)));

File: UVa/300/369.cpp
#include <bits/stdc++.h>
#include "combinatorics.h"
#include "header.h"

int main() {
	int n, m;
	while (cin >> n >> m && (n | m) != 0)
		print("%d things taken %d at a time is %lld exactly.", n, m, binom(n, m));
}
